librz/include/rz_util/rz_table.h

```diff
@@ -89,11 +89,20 @@ RZ_API RzTable *rz_table_pop(RzTable *t);
 RZ_API void rz_table_fromjson(RzTable *t, const char *csv);
 RZ_API void rz_table_fromcsv(RzTable *t, const char *csv);
 RZ_API char *rz_table_tohtml(RzTable *t);
-RZ_API void rz_table_transpose(RzTable *t);
+RZ_API RzTable *rz_table_transpose(RzTable *t);
```

Comment:
Please move the Doxygen documentation to the C file instead of the header, and use `\brief` syntax, like the rest of Rizin code:
- https://github.com/rizinorg/rizin/blob/dev/DEVELOPERS.md#documentation
- https://github.com/rizinorg/rizin/blob/dev/librz/bin/dbginfo.c#L19